core/commands/validate.go

```diff
@@ -0,0 +1,25 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Date Command
+func ValidateCommand() (*cobra.Command, error) {
+	// creating new command
+	// using localizer to access default text by ID provided in locals
+	cmd := &cobra.Command{
+		Use:     "validate",
+		Short:   "validate custom commands",
+		Long:    "validates the custom commands with the defined rules",
+		Example: "host charmil validate",
+		Args:    cobra.ExactArgs(0),
+		// SilenceUsage: true,
```

Comment:
No comments

Reply:
I see alot of comments that I left in from copying the date command, will clean this up.